charts/victoria-metrics-agent/values.yaml

```diff
@@ -4,6 +4,22 @@
 
 replicaCount: 1
 
+## Deployment or StatefulSet
+# Deployment ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
+# StatefulSet ref:  https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+kind: Deployment
+
+## WARN: If kind is Deployment that uses
+deploymentStrategy: {}
```

Comment:
don't  need to add comment if keys will already contain type of strategy eg.
```
updateStrategy
 statefulset:
 deployemnt:
 ```

Reply:
replaced adn added ref to documentation